api/v6/products.thrift

```diff
@@ -35,11 +35,13 @@ struct Product {
   2: string            endpoint,
   3: string            displayedName_b64,
   4: string            description_b64,
-  5: bool              connected,         // True only if database status is OK.
-                                          // !DEPRECATED FLAG databaseStatus is used to get the status of the database.
-  6: bool              accessible,        // Indicates whether the current user can access this product.
-  7: bool              administrating     // Indicates that the current user can administrate the product.
-  8: shared.DBStatus   databaseStatus     // Indicates the database backend status.
+  5: bool              connected,            // True only if database status is OK.
+                                             // !DEPRECATED FLAG databaseStatus is used to get the status of the database.
+  6: bool              accessible,           // Indicates whether the current user can access this product.
+  7: bool              administrating,       // Indicates that the current user can administrate the product.
+  8: shared.DBStatus   databaseStatus,       // Indicates the database backend status.
+  9: i64               runCount,             // Number of runs in the product.
+  10: string           latestUpdatedRunDate, // Latest date from the runs when the run in the product was updated.
```

Comment:
Is it good that this is a `string`? Are we sending dates as strings elsewhere too?

Reply:
Yes, we are sending dates as strings in multiple places:

https://github.com/Ericsson/codechecker/blob/b9987109e6dac5aff4babf9bac47a34d3ad539e4/api/v6/report_server.thrift#L115
https://github.com/Ericsson/codechecker/blob/b9987109e6dac5aff4babf9bac47a34d3ad539e4/api/v6/report_server.thrift#L130
https://github.com/Ericsson/codechecker/blob/b9987109e6dac5aff4babf9bac47a34d3ad539e4/api/v6/report_server.thrift#L163-L164